vscode/src/commands.ts

```diff
@@ -84,6 +84,23 @@ const commandsMap: (state: ExtensionState) => {
         console.error("Could not start the server", e);
       }
     },
+    "konveyor.getToken": async (): Promise<string | undefined> => {
+      let token = await state.extensionContext.secrets.get("konveyor.token");
+
+      if (!token) {
+        // TODO(djzager): Update this to get the user's password
+        // once that logic is implemented in Hub/SolutionServer
```

Comment:
@jortel 

I think this logic is implemented in the hub already (it's just using the normal auth flow, nothing solution server specific)